kahuna/public/js/usage-rights/usage-rights-editor.css

```diff
@@ -35,3 +35,7 @@
 .ure__multiple-rights-warning {
     color: orange;
 }
+
+.ure--full-width-field {
```

Comment:
`--` is a modifier state. Not sure you even need to namespace with `ure` here, but if you want to, you could use either `ure-full-width-field` or `ure__full-width-field`.